src/rust-crypto/CrossSigningIdentity.ts

```diff
@@ -123,6 +138,9 @@ export class CrossSigningIdentity {
      * (If secret storage is *not* configured, we assume that the export will happen when it is set up)
      */
     private async exportCrossSigningKeysToStorage(): Promise<void> {
-        // TODO
+        const exported: RustSdkCryptoJs.CrossSigningKeyExport = await this.olmMachine.exportCrossSigningKeys();
+        this.secretStorage.store("m.cross_signing.master", exported.masterKey!);
+        this.secretStorage.store("m.cross_signing.self_signing", exported.self_signing_key!);
+        this.secretStorage.store("m.cross_signing.user_signing", exported.userSigningKey!);
```

Comment:
how do we know that these ! assertions are safe?

Reply:
It's a private function that we call when we know we have secrets.
Do you think I should be lenient and just log if it's not stored?

Reply:
Updated, and now logging an error if secret parts unknown